addons/default/visiosoft/cats-module/src/Category/CategoryRepository.php

```diff
@@ -50,7 +50,15 @@ public function getCatById($id)
 
     public function getSubCatById($id)
     {
-        return $this->model->where('parent_category_id', $id)->where('deleted_at', null)->orderBy('sort_order')->get();
+        $orderBy = $this->model->getParentsCount($id) >= 1 ? 'name' : 'sort_order';
+        return $this->model->newQuery()
```

Comment:
it wasn't working multi language cats? @diashalabi 

@spektra2147 any idea

Reply:
yeah `->where('cats_category_translations.locale', config('app.locale'))`

Reply:
model should take care of translation queries. Beacause in next version ryan putting translation in 1 json column. Also We should be duplicate ourself. There was problem in model for translations and we forked streams-platform @diashalabi @spektra2147 


https://github.com/openclassify/streams-platform/commit/2222954d72abdf2b47411be83b17d188ab2374c8

Reply:
If we want to fix something in model for translation you can edit streams platform. It will affect all queries then

Reply:
are we not going to use ryan's new version?

Reply:
we're going to use it and we shouldn't write our own sql queries for translations